workflows/quickstart.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START workflows_quickstart]
+import {WorkflowsClient} from '@google-cloud/workflows';
+
+async function main(projectId: string, location: string) {
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'my-project';
+  // const location = 'us-central1';
+  const client = new WorkflowsClient();
+  async function listWorkflows() {
+    const [workflows] = await client.listWorkflows({
+      parent: client.locationPath(projectId, location),
+    });
+    for (const workflow of workflows) {
+      console.info(`name: ${workflow.name}`);
+    }
+  }
+  listWorkflows();
+}
+// [END workflows_quickstart]
+
+process.on('unhandledRejection', (err: Error) => {
+  console.error(err.message);
+  process.exitCode = 1;
+});
+main(...(process.argv.slice(2) as [string, string]));
```

Comment:
I would personally pull the invocation of main into the sample itself, because I'm not confident users will understand that they need to call the method, so:


```js
const [projectId, location] = process.argv.slice(2);
// [START workflows_quickstart]
main(projectId, location);
// [END workflows_quickstart]
process.on('unhandledRejection', (err: Error) => {
  console.error(err.message);
  process.exitCode = 1;
});
```

Or, alternatively:

```js
const [projectId, location] = process.argv.slice(2);
// [START workflows_quickstart]
main(projectId, location).catch((err: Error) = {
  console.error(err.message);
  process.exitCode = 1;
});
// [END workflows_quickstart]
```